### PR TITLE
Fix Invalid use of BasicClientConnManager: connection still allocated.

### PR DESCRIPTION
**Pull request by pmouawad**
When you face an error in some download the plugin is completely broken with :
2017-12-02 21:30:36,492 ERROR o.j.r.PluginManager: Failed to download jna
java.lang.IllegalStateException: Invalid use of BasicClientConnManager: connection still allocated.
Make sure to release the connection before allocating another one.
	at org.apache.http.util.Asserts.check(Asserts.java:34) ~[httpcore-4.4.7.jar:4.4.7]
	at org.apache.http.impl.conn.BasicClientConnectionManager.getConnection(BasicClientConnectionManager.java:163) ~[httpclient-4.5.3.jar:4.5.3]
	at org.apache.http.impl.conn.BasicClientConnectionManager$1.getConnection(BasicClientConnectionManager.java:145) ~[httpclient-4.5.3.jar:4.5.3]
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:422) ~[httpclient-4.5.3.jar:4.5.3]
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835) ~[httpclient-4.5.3.jar:4.5.3]
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83) ~[httpclient-4.5.3.jar:4.5.3]
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56) ~[httpclient-4.5.3.jar:4.5.3]
	at org.jmeterplugins.repository.JARSourceHTTP.execute(JARSourceHTTP.java:376) ~[jmeter-plugins-manager-0.16.jar:?]
	at org.jmeterplugins.repository.JARSourceHTTP.getJAR(JARSourceHTTP.java:284) ~[jmeter-plugins-manager-0.16.jar:?]
	at org.jmeterplugins.repository.PluginManager.applyChanges(PluginManager.java:163) [jmeter-plugins-manager-0.16.jar:?]
	at org.jmeterplugins.repository.PluginManagerDialog$4.run(PluginManagerDialog.java:226) [jmeter-plugins-manager-0.16.jar:?]

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <groupId>kg.apc</groupId>
     <artifactId>jmeter-plugins-manager</artifactId>
-    <version>0.15</version>
+    <version>0.17</version>
```
Comment (undera) [inline pom.xml]: The version choice is made on release, it is not meant to be changed by every PR. Just for the future info...

**Comment by undera** [comment]
Please review unit test failure

**Comment by pmouawad** [comment]
Hi @undera ,
I don't understand the failures. 
If I run tests in Eclipse it works. If I do using same command as  travis it fails, but even if I comment all my changes it still fails.

Regards

**Comment by undera** [comment]
Yes, looks like something broke in the environment for these tests. I will figure it out and will let you know, then we'll trigger tests for your PR again.

**Comment by undera** [comment]
Please merge from master now, it should fix the tests.

**Comment by pmouawad** [comment]
PR merged as requested

**Pull request merged**